storm-core/test/clj/backtype/storm/nimbus_test.clj

```diff
@@ -846,16 +846,6 @@
                   NIMBUS-SLOTS-PER-TOPOLOGY 8}]
     (letlocals
       (bind topology (thrift/mk-topology
-                        {"1" (thrift/mk-spout-spec (TestPlannerSpout. true) :parallelism-hint 0 :conf {TOPOLOGY-TASKS 1})}
```

Comment:
I agree on this.

However, I still don't understand why changing the expected exception type does not "fix" the test... If I put `thrown? IllegalArgumentException` instead of `thrown? InvalidTopologyException` the test does not pass but fails. Can you explain?